### PR TITLE
#HT-192 : Create the `Checkbox` Component

### PR DESCRIPTION
**Pull request by pranjal5399**

### CODE REVIEW

File: packages/console/src/types/index.tsx [hunk 1]
```diff
@@ -138,3 +139,13 @@ export type SelectColor =
...
+
+export type CheckboxSize = "small" | "medium";
+
+export type CheckboxColor =
+    | "primary"
+    | "secondary"
+    | "success"
+    | "error"
+    | "info"
+    | "warning";
```
Comment (joelerego) [inline packages/console/src/types/index.tsx:151]: Can we merge all Color types into a single type? Can it be done for size too?

File: packages/console/src/nodes/Checkbox.tsx [hunk 2]
```diff
@@ -0,0 +1,149 @@
...
+                                type: "text",
+                                size: "small",
+                                help: "Label for the checkbox",
+                            },
+                            {
+                                id: "disabled",
+                                title: "Disabled",
+                                type: "switch",
+                                size: "small",
+                                help: "Determines wether the checkbox is disabled, or not.",
```
Comment (joelerego) [inline packages/console/src/nodes/Checkbox.tsx:108]: Typo here (`wether`)

File: packages/console/src/nodes/Checkbox.tsx [hunk 3]
```diff
@@ -0,0 +1,149 @@
...
+                                    { value: "small", title: "Small" },
+                                    { value: "medium", title: "Medium" },
+                                ],
+                            },
+                            {
+                                id: "checked",
+                                title: "Checked",
+                                type: "switch",
+                                size: "small",
+                                help: "Determines wether the checkbox is checked by default, or not.",
```
Comment (joelerego) [inline packages/console/src/nodes/Checkbox.tsx:94]: Typo here (`wether`)

**Review by joelerego — CHANGES_REQUESTED** [review]

File: packages/console/src/types/index.tsx (hunk 1 above)
Comment (pranjal5399) [reply packages/console/src/types/index.tsx:151]: Done

**Review by itssamuelrowe — APPROVED** [review]

**Review by joelerego — APPROVED** [review]

**Pull request merged**